sympy/physics/optics/tests/test_medium.py

```diff
@@ -29,8 +29,11 @@ def test_medium():
     assert m4.refractive_index < m1.refractive_index
     assert m4 < m1
     m5 = Medium('m5', permittivity=710*10**(-12)*s**4*A**2/(m**3*kg), n=1.33)
-    assert simplify(m5.intrinsic_impedance - 6.24845417765552*kg*m**2/(A**2*s**3)) == 0
+    assert abs(m5.intrinsic_impedance - 6.24845417765552*kg*m**2/(A**2*s**3)) \
+                < 1e-12*kg*m**2/(A**2*s**3)
     assert abs(m5.speed - 225407863.157895*m/s) < 1e-6*m/s
-    assert simplify(m5.refractive_index - 1.33000000000000) == 0
-    assert simplify(m5.permittivity - 7.1e-10*A**2*s**4/(kg*m**3)) == 0
-    assert simplify(m5.permeability - 2.77206575232851e-8*kg*m/(A**2*s**2)) == 0
+    assert abs(m5.refractive_index - 1.33000000000000) < 1e-12
+    assert abs(m5.permittivity - 7.1e-10*A**2*s**4/(kg*m**3)) \
```

Comment:
Could you use verify numerically again: `assert verify_numerically(m5.permittiity.as_coeff_Mul()[0], 7.1e-10)` and similar for the following? Mostly nits...if you don't want to that's fine.